src/pybind/mgr/cephadm/services/nfs.py

```diff
@@ -51,7 +51,7 @@ def generate_config(self, daemon_spec: CephadmDaemonSpec[NFSServiceSpec]) -> Tup
 
         # create the keyring
         user = f'{daemon_type}.{daemon_id}'
-        entity = utils.name_to_config_section(user)
+        entity = self.get_auth_entity(daemon_id)
```

Comment:
```suggestion
        daemon_name = f'{daemon_type}.{daemon_id}'
        entity = self.get_auth_entity(daemon_name)
```

Reply:
this would produce: `client.nfs.nfs.{daemon_id}` ... ?